src/views/InvolvementPanel/InvolvementPanel.js

```diff
@@ -18,6 +19,10 @@ export default class InvolvementPanel extends React.Component {
         text: 'Volunteer',
         view: <VolunteerFormView />
       },
+      TEACHER: {
```

Comment:
Now that we have 4 form links on the switcher bar, how does it look as you reduce the width of the window? Does the text wrap and look odd?